[START]---------------------------------------
Timestamp: 11/19/2016 5:00:56 AM

Message:    at System.Web.Http.HostAuthenticationFilter.GetAuthenticationManagerOrThrow(HttpRequestMessage request)
   at System.Web.Http.HostAuthenticationFilter.<AuthenticateAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.PassiveAuthenticationMessageHandler.<SendAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()

Category: General

Priority: -1

EventId: 0

Severity: Error

Title:

Machine: ALEX-PC

App Domain: /LM/W3SVC/3/ROOT-1-131240052435453460

ProcessId: 19160

Process Name: C:\Program Files (x86)\IIS Express\iisexpress.exe

Thread Name: 

Win32 ThreadId:14656

Extended Properties: 
[END]-----------------------------------------
[START]---------------------------------------
Timestamp: 11/19/2016 5:00:58 AM

Message: Tracing to LogSource 'General' failed. Processing for other sources will continue. See summary information below for more information. Should this problem persist, stop the service and check the configuration file(s) for possible error(s) in the configuration of the categories and sinks.


Summary for Enterprise Library Distributor Service:
======================================
--> 
Message: 
Timestamp: 11/19/2016 5:00:56 AM
Message:    at System.Web.Http.HostAuthenticationFilter.GetAuthenticationManagerOrThrow(HttpRequestMessage request)
   at System.Web.Http.HostAuthenticationFilter.<AuthenticateAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.PassiveAuthenticationMessageHandler.<SendAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
Category: General
Priority: -1
EventId: 0
Severity: Error
Title:
Machine: ALEX-PC
App Domain: /LM/W3SVC/3/ROOT-1-131240052435453460
ProcessId: 19160
Process Name: C:\Program Files (x86)\IIS Express\iisexpress.exe
Thread Name: 
Win32 ThreadId:14656
Extended Properties: 
--> MachineName: ALEX-PC
--> TimeStamp: 11/19/2016 5:00:58 AM
--> FullName: Microsoft.Practices.EnterpriseLibrary.Logging, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
--> AppDomainName: /LM/W3SVC/3/ROOT-1-131240052435453460
--> WindowsIdentity: FMP\Alejandro.Clavijo

Exception Information Details:
======================================
Exception Type: System.Net.Mail.SmtpException
StatusCode: GeneralFailure
Message: Failure sending mail.
Data: System.Collections.ListDictionaryInternal
TargetSite: Void Send(System.Net.Mail.MailMessage)
HelpLink: NULL
Source: System
HResult: -2146233088

StackTrace Information Details: 
======================================
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at Microsoft.Practices.EnterpriseLibrary.Logging.Configuration.EmailMessage.SendMessage(MailMessage message)
   at Microsoft.Practices.EnterpriseLibrary.Logging.Configuration.EmailMessage.Send()
   at Microsoft.Practices.EnterpriseLibrary.Logging.TraceListeners.EmailTraceListener.TraceData(TraceEventCache eventCache, String source, TraceEventType eventType, Int32 id, Object data)
   at Microsoft.Practices.EnterpriseLibrary.Logging.LogSource.TraceData(TraceEventType eventType, Int32 id, LogEntry logEntry, TraceListenerFilter traceListenerFilter, TraceEventCache traceEventCache, ReportTracingError reportError)

Exception Information Details:
======================================
Exception Type: System.Net.WebException
Status: ConnectFailure
Response: NULL
Message: Unable to connect to the remote server
Data: System.Collections.ListDictionaryInternal
TargetSite: System.Net.Sockets.Socket GetConnection(System.Net.PooledStream, System.Object, Boolean, System.Net.IPAddress ByRef, System.Net.Sockets.Socket ByRef, System.Net.Sockets.Socket ByRef)
HelpLink: NULL
Source: System
HResult: -2146233079

StackTrace Information Details: 
======================================
   at System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress& address, Socket& abortSocket, Socket& abortSocket6)
   at System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   at System.Net.PooledStream.Activate(Object owningObject, GeneralAsyncDelegate asyncCallback)
   at System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpTransport.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.GetConnection()
   at System.Net.Mail.SmtpClient.Send(MailMessage message)

Exception Information Details:
======================================
Exception Type: System.Net.Sockets.SocketException
ErrorCode: 10061
Message: No connection could be made because the target machine actively refused it 127.0.0.1:25
SocketErrorCode: ConnectionRefused
NativeErrorCode: 10061
Data: System.Collections.ListDictionaryInternal
TargetSite: Void DoConnect(System.Net.EndPoint, System.Net.SocketAddress)
HelpLink: NULL
Source: System
HResult: -2147467259

StackTrace Information Details: 
======================================
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)

Category: 

Priority: -1

EventId: 6352

Severity: Error

Title:

Machine: ALEX-PC

App Domain: /LM/W3SVC/3/ROOT-1-131240052435453460

ProcessId: 19160

Process Name: C:\Program Files (x86)\IIS Express\iisexpress.exe

Thread Name: 

Win32 ThreadId:14656

Extended Properties: 
[END]-----------------------------------------
